src/authorization/Rules.ts

```diff
@@ -10,4 +10,5 @@ export default {
     "APP_UNPACK_ORDER": "COMMON_ADMIN",
     "APP_RECYCLE_ORDER": "COMMON_ADMIN",
     "APP_STOREFULFILLMENT_ADMIN": "STOREFULFILLMENT_ADMIN",
+    "APP_SHIP_ORDER": "COMMON_ADMIN"
```

Comment:
permission name should also include that this is "forceful" like "APP_FORCE_SHIP_ORDER" because for normal orders anyone is allowed to ship them